### PR TITLE
Add text wireshark support

### PR DESCRIPTION
**Pull request by gpotter2**
- adds tshark shortcut

### CODE REVIEW

**Comment by gpotter2** [comment]
Ready to be reviewed

File: scapy/utils.py [hunk 1]
```diff
@@ -1365,12 +1365,18 @@ def import_hexcap():
...
     """Run wireshark on a list of packets"""
     f = get_temp_file()
     wrpcap(f, pktlist)
-    with ContextManagerSubprocess("wireshark()", conf.prog.wireshark):
-        subprocess.Popen([conf.prog.wireshark, "-r", f])
+    if text_mode:
+        prog = "tshark"
+        args = [conf.prog.tshark, "-V", "-r"]
+    else:
+        prog = "wireshark"
```
Comment (p-l-) [inline scapy/utils.py]: Certainly not! Please use `conf.prog.xxx`.

Comment (gpotter2) [reply scapy/utils.py]: See just below :)

`prog` variable here is only used in the context manager as the name of the util that fails if an error happens

Comment (p-l-) [reply scapy/utils.py]: Oh of course yes!

**Comment by gpotter2** [comment]
@guedou @p-l- Please review

**Comment by guedou** [comment]
Is it possible to test this?

**Comment by gpotter2** [comment]
@guedou I’ll try to add a test. Should be possible

**Pull request merged**